steam/ext/csgo/enums.py

```diff
@@ -98,7 +98,25 @@ class ItemOrigin(IntEnum):
     LevelUpReward             = 24
     Max                       = 25
 
+class PerMapRankEnum(IntEnum):
+    MapID                     = 1
+    RANK_ID                   = 2
+    WINS                      = 3
```

Comment:
```suggestion
    MapID                     = 1
    RANK_ID                   = 2
    WINS                      = 3
```
Consistent naming here is required

Reply:
Done.